core/server/common/src/main/java/alluxio/master/StateLockManager.java

```diff
@@ -247,6 +260,9 @@ private void deactivateInterruptCycle() {
    * Scheduled routine that interrupts waiters/holders of shared lock.
    */
   private void waiterInterruptRoutine() {
+    LOG.debug("Interrupt-cycle interrupting %d waiters/holders: {%s}",
+        Thread.currentThread().getId(), mSharedWaitersAndHolders.size(), mSharedWaitersAndHolders
+            .stream().map((th) -> Long.toString(th.getId())).collect(Collectors.joining(",")));
```

Comment:
this may not be accurate, since the real interrupt happens in the loop below.

Reply:
Bumped it down for consistent logging.